src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/RequestDelegateValidateGeneratedFormCode.generated.txt

```diff
@@ -345,6 +345,7 @@ namespace Microsoft.AspNetCore.Http.Generated
 
     }
 
+
```

Comment:
I could collapse the variables on to one line (e.g. `{{var1}}{{var2}}` to get rid of the errant whitespace. Another alternative would be to cut this method over to using a CodeWriter and then call into other CodeWriters (an approach we've used elsewhere).

If we want to do this, then it should be a separate PR though since it's a lot of churns in that central emission code by itself.